ddp-workspace/projects/ddp-sdk/src/lib/services/activity/activityQuestionConverter.service.ts

```diff
@@ -63,6 +63,7 @@ export class ActivityQuestionConverter {
         questionBlock.tooltip = questionJson.tooltip;
         questionBlock.readonly = questionJson.readonly;
         questionBlock.displayNumber = displayNumber;
+        questionBlock.columnSpan = questionJson.columnSpan ?? 1;
```

Comment:
We default to 1 column for questitions

Reply:
good point.

Reply:
Actually, should it default 2 columns instead? i.e., one for the prompt, one of the the field(s)?

Reply:
No. We hide any question's prompt for the tabular view. So 1 column for 1 question is alright. 
Composite question is an exception here, we need to set `columnSpan` explicitly for it 
( `columnSpan = children.length + (doWeNeedCompositeQuestionPrompt ? 1 : 0)` )